expertiza/expertiza
# E1797: [TDD] Timestamps for students' submissions

**Credentials for student :-**

User Id : student6603, student6604
Password : password

**Testing Methodology:**

Case1:   Timestamps for the deadlines should be viewed when no submissions are done.

How to Test:
1. Login as a student
2. Select an Assignment which is active
3. The timeline should display just the deadlines with timestamps in chronological order when no work is submitted.

Case2:   Timestamps for the submissions should be viewed for submitted Hyperlink  and deleted hyperlink on timeline

How to Test:
1. Login as a student
2. Select an Assignment which is active
3. Select “Your Work”
4. Submit a Hyperlink
5. The Hyperlink should appear on the Timeline with the timestamp as “Submit Hyperlink”. The hyperlink would be redirected      to the page when clicked.
6. Remove the hyperlink from “Your Work”
7. The timeline should “ Remove Hyperlink”  with the timestamp

Case3:   Timestamps for the submissions should be viewed for submitted file  and deleted  file on timeline

How to Test:
1. Login as a student
2. Select an Assignment which is active
3. Select “Your Work”
4. Submit a file
5. The “Submit File” should appear on the Timeline with the timestamp
6. Delete the submitted file from “Your Work”
7. The timeline should show a timestamp with “Remove File”

Case4:   Timestamps for the  peer review done should appear on timeline

How to Test:
1. Login as a student
2. Select an Assignment which is active
3. Select “Other’s Work ”
4. Choose a topic and review
5.  The timeline should show a timestamp with “Round 1 Review”  or “Round 2 review” depending on the Round  of the review

Case5:   Timestamps for the  author feedback done should appear on timeline

How to Test:
1. Login as a student
2. Select an Assignment which is active
3. Select Your Scores – Alternate View
4. Click on Show review and then “Give Feedback”
5. The timeline should show a timestamp with the hyperlink to the feedback link. Clicking the Feedback link should be redirect the user to the feedback page.
